src/pipelines/master_pipeline.py

```diff
@@ -292,6 +303,14 @@ def get_args(
     if len(features) == 0:
```

Comment:
The default-setting code below doesn't keep this condition from firing and quitting if only W2V_MIN/MAX/etc is selected but W2V_APPEND/etc isn't